TAO/orbsvcs/orbsvcs/AV/AVStreams_i.cpp

```diff
@@ -2025,11 +2025,8 @@ TAO_StreamEndPoint::request_connection (AVStreams::StreamEndPoint_ptr /*initiato
       AVStreams::streamQoS network_qos;
       if (qos.length () > 0)
         {
-         if (TAO_debug_level > 0)
-         {
-            ORBSVCS_DEBUG ((LM_DEBUG,
-                        "QoS is Specified\n"));
-         }
+          if (TAO_debug_level > 0)
```

Comment:
Why only change this one? I prefer the original code

Reply:
The compiler warns for "misleading" indents -- when code that belongs to the same block is not indented the same way.

Here's the original code with additional info:
```
      if (qos.length () > 0)                      // 6 spaces
        {                                         // 8 spaces
         if (TAO_debug_level > 0)                 // 9 spaces
         {                                        // 9 spaces
            ORBSVCS_DEBUG ((LM_DEBUG,             // 12 spaces
                        "QoS is Specified\n"));   // 24 spaces
         }                                        // 9 spaces

          int const result = this->translate_qos (qos, network_qos); // 10 spaces
```

We can change it to have `{}` and also be indented correctly.